applications/visit/app.json

```
@@ -0,0 +1,93 @@
+        "execSystemExecDir": "${JobWorkingDir}",
+        "execSystemInputDir": "${JobWorkingDir}",
+        "execSystemOutputDir": "${JobWorkingDir}/output",
+        "execSystemLogicalQueue": "small",
+        "archiveSystemId": "cloud.data",
+        "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+        "archiveOnAppError": true,
+        "isMpi": false,
+        "mpiCmd": null,
+        "cmdPrefix": null,
+        "parameterSet": {
+            "appArgs": [],
+            "containerArgs": [],
+            "schedulerOptions": [],
+            "envVariables": [
+                {
+                    "key": "desktop_resolution",
+                    "value": "1280x800",
+                    "description": "Set the desktop screen size for your visualization session.",
+                    "inputMode": "REQUIRED",
+                    "notes": {
+                        "label": "Desktop Resolution",
+                        "enum_values": [
+                            {
+                                "800x600": "800x600"
+                              },
+                              {
+                                "1280x800": "1280x800"
+                              },
+                              {
+                                "1920x1080": "1920x1080"
+                              },
+                              {
+                                "1920x1200": "1920x1200"
+                              },
+                              {
+                                "1600x1200": "1600x1200"
+                              },
+                              {
+                                "2560x1600": "2560x1600"
+                              }
+                        ]
+                    }
+                }
+            ]        
+        },
+        "fileInputs": [
+            {
+                "name": "Working Directory",
+                "description": "The directory containing the files that you want to work on. This directory and its files will be copied to where your interactive session runs. You can drag the link for the directory from the Data Browser on the left, or click the 'Select Input' button and then select the directory.",
+                "inputMode": "REQUIRED",
+                "autoMountLocal": true,
+                "sourceUrl": null,
+                "targetPath": "workingDirectory"
+            }
+        ],
+        "fileInputArrays": [],
+        "nodeCount": 1,
+        "coresPerNode": 56,
+        "memoryMB": 192,
+        "maxMinutes": 120,
+        "subscriptions": [],
+        "tags": []
+    },
+    "tags": [
+        "portalName: DesignSafe"
+    ],
+    "notes": {
+        "icon": "VisIt",
+        "label": "VisIt",
+        "helpUrl": "https://www.designsafe-ci.org/rw/user-guides/tools-applications/visualization/visit/",
+        "hideNodeCountAndCoresPerNode": false,
+        "category": "Visualization"
```

we'll need the isInteractive flag
```suggestion
        "category": "Visualization",
        "isInteractive": true,
```